content/en/docs/Reference/configuration.md

```diff
@@ -679,7 +679,7 @@ Enables the embedded registry in k3s. Accepted: `true`, `false`.
 
 This section allows you to configure the k3s agent instances.
 
-### `k3s-agent.enable`
+### `k3s-agent.enabled`
```

Comment:
https://github.com/kairos-io/provider-kairos/blob/6f7c6df760e4b52351eb69a60a1f520f192c7267/internal/provider/config/config.go#L109

Reply:
This is the same struct for both settings (here and the one above)